handbook/engineering/releases/index.md

```diff
@@ -128,6 +128,9 @@ This is because:
 
 In the future, we may introduce continuous releases if these issues become surmountable.
 
-### Weekend releases
 
-If the 20th falls on a non-working day, the release captain will move the release forward to the first working day. The calendar events will reflect this.
+### Important dates
+
+The release branch will be cut on the last working day at 10am PST before the 20th of each month. If the 20th falls on a non-working day, the release captain will move the release forward to the first working day, as well as cutting the release branch the day before. The calendar events will reflect this.
+
+eg. The 20th of a given month falls on a Sunday, therefore the release will be moved forward to the first working day (the 18th) and the branch will cut the day before (the 17th). 
```

Comment:
q (was absent for the meeting): are we moving away from the 1-day release we were aiming for before? (https://github.com/sourcegraph/sourcegraph/pull/22773#issuecomment-878801279)

Reply:
For time time being, yes. Until such time as CI doesn't take 25-30 minutes to run each time, we agreed it's better to spread this out. I've also encountered issues in the past where CI has broken on the release branch and it causes undue stress on the release captain as distribution don't own any of the tests or code, and it isn't always easy to get people in a different timezone to fix things.

Reply:
thank you! (just watched the meeting and heard the discussion, agreed)